system/jhtree/jhtree.cpp

```diff
@@ -351,6 +351,8 @@ class jhtree_decl CKeyLevelManager : implements IKeyManager, public CInterface
 {
 protected:
     KeyStatsCollector stats;
+    KeyStatsCollector nullStats;
+    KeyStatsCollector *conditionalStats = nullptr;
```

Comment:
conditionalStats isn't a great name - it begs the question what is it conditional all.  Maybe activeStats or another variant.